### DISCUSSION TITLE
Add an option to use old UI structure and/or sorting

### DISCUSSION THREAD

**Discussion by crumpledmint** [ppy/osu#28490] [Ideas] [answered]
One of the most complained about areas of lazer is UI and although I understand that new implementation of it is being worked on, I think that it won't satisfy the conservative part of the player base. Why not just add an option to use the old UI structure and/or sorting/grouping for those and save the current/next version of current UI as a default option?

**Comment by peppy** [comment, marked as answer]
the goal is to provide a structure which matches expectations, yes.
